website/source/docs/builders/vagrant.html.md

```diff
@@ -0,0 +1,104 @@
+The Vagrant builder is intended for building new boxes from already-existing
+boxes. Your source should be a URL or path to a .box file or a Vagrant Cloud
+box name such as `hashicorp/precise64`. This builder is not currently intended
+to work with an already-set-up Vagrant environment.
+
+Packer will not install vagrant, nor will it install the underlying
+virtualization platforms or extra providers; We expect when you run this
+builder that you have already installed what you need.
+
+By default, this builder will initialize a new Vagrant workspace, launch your
+box from that workspace, provision it, call `vagrant package` to package it
+into a new box, and then destroy the original box. Please note that vagrant
+will _not_ remove the box file from your system (we don't call
+`vagrant box remove`.
+
+You can change the behavior so that the builder doesn't destroy the box by
+setting the `teardown_method` option. You can change the behavior so the builder
+doesn't package it (not all provisioners support the `vagrant package` command)
+by setting the `skip package` option.
+
+Required:
+
+-    `source_box` (string) - URL of the vagrant box to use, or the name of the
+    vagrant box. `hashicorp/precise64`, `./mylocalbox.box` and
+    `https://boxes.company.com/my-company.box` are all valid source boxes. If your
```

Comment:
Not sure about how other examples are defined in docs but `example.com` might be better here since it's the standard example domain?